extensions/amp-list/0.1/test/test-amp-list.js

```diff
@@ -174,77 +133,14 @@ describes.realWin('amp-list component', {
     });
 
     it('should fetch and render', () => {
-      const items = [
-        {title: 'Title1'},
-      ];
+      const items = [{title: 'Title1'}];
       const itemElement = doc.createElement('div');
       const rendered = expectFetchAndRender(items, [itemElement]);
       return list.layoutCallback().then(() => rendered).then(() => {
         expect(list.container_.contains(itemElement)).to.be.true;
       });
     });
 
-    describe('Viewer render template', () => {
-      const allowViewerRenderTemplate = 'allow-viewer-render-template';
-      beforeEach(() => {
-        win.document.documentElement.setAttribute(
-            allowViewerRenderTemplate, true);
-      });
-
-      afterEach(() => {
-        win.document.documentElement.removeAttribute(
-            allowViewerRenderTemplate);
-      });
-
-      it('should proxy rendering to viewer', () => {
-        const setupAMPCors = sandbox.spy(xhrUtils, 'setupAMPCors');
-        const fromStructuredCloneable =
-            sandbox.spy(xhrUtils, 'fromStructuredCloneable');
-        const verifyAmpCORSHeaders =
-            sandbox.spy(xhrUtils, 'verifyAmpCORSHeaders');
-        const resp = {'html': '<div>Rendered template</div>'};
-        const itemElement = doc.createElement('div');
-        const rendered = expectViewerProxiedFetchAndRender(resp, itemElement);
-        return list.layoutCallback().then(() => rendered).then(() => {
-          expect(list.container_.contains(itemElement)).to.be.true;
-          sinon.assert.callOrder(
-              setupAMPCors, fromStructuredCloneable, verifyAmpCORSHeaders);
-        });
-      });
-
-      it('should error if viewer does not define response data', () => {
-        win.document.documentElement.setAttribute(
-            'allow-viewer-render-template', true);
-        viewerMock.expects('hasCapability')
-            .withExactArgs('viewerRenderTemplate').returns(true);
-        viewerMock.expects('sendMessageAwaitResponse').withExactArgs(
-            'viewerRenderTemplate',
-            {
-              'ampComponent': {
-                'errorTemplate': {'payload': null, 'type': 'amp-mustache'},
-                'src': 'https://data.com/list.json',
-              },
-              'successTemplate': {
-                'payload': '<template xmlns="http://www.w3.org/1999/xhtml">{{template}}</template>',
-                'type': 'amp-mustache',
-              },
-              'type': 'amp-list',
-              'originalRequest': {
-                'init': {
-                  'headers': {'Accept': 'application/json'},
-                  'method': 'GET',
-                  'requireAmpResponseSourceOrigin': false,
-                },
-                'input': 'https://data.com/list.json',
-              },
-            }).returns(Promise.resolve({})).once();
```

Comment:
@alabiaga Let's move these viewer API expectations to a unit test for `SsrTemplateHelper`.